internal/olm/operator/registry/index/registry_pod_test.go

```diff
@@ -114,6 +132,11 @@ var _ = Describe("RegistryPod", func() {
 						ImageTag: "quay.io/example/example-operator-bundle:1.0.1",
 						AddMode:  SemverBundleAddMode,
 					},
+					BundleItem{
+						ImageTag: "localhost/example-operator-bundle:1.0.1",
+						AddMode:  SemverBundleAddMode,
+						SkipTLS:  true,
```

Comment:
Like was said earlier, this field should be on RegistryPod.

Reply:
Done.